tests/main/snap-cli-no-managers/task.yaml

```diff
@@ -17,6 +17,6 @@ execute: |
   check_imports /usr/bin/snap
 
   # snap-bootstrap is shipped only on ubuntu (except for 14.04)
-  if [[ "$SPREAD_SYSTEM" = ubuntu-2* ]] ||  [[ "$SPREAD_SYSTEM" = ubuntu-18.04-* ]] || [[ "$SPREAD_SYSTEM" = ubuntu-16.04-* ]]; then
+  if [[ "$SPREAD_SYSTEM" = ubuntu-2* ]] || os.query is-bionic || os.query is-xenial; then
```

Comment:
Same question about mixing old-style SPREAD_SYSTEM check.

Reply:
In this case, I still need to check how the make a command to determine if a version is newer than 20.04, I'll try to add that to the next PR